### DISCUSSION TITLE
SKIPPED: Windows host tools are not available yet.

### DISCUSSION THREAD

**Discussion by zafersn** [zephyrproject-rtos/zephyr#63914] [Q&A]
I am following the getting started guide https://docs.zephyrproject.org/latest/develop/getting_started/index.html to install zephyr workspace.

but I got this error when I ran setup.cmd command at step [Install Zephyr SDK](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#install-zephyr-sdk) paragraph 4.
 command: setup.cmd
```

(.venv) c:\netfeasaproject\zephyr-sdk-0.16.3>setup.cmd
Zephyr SDK 0.16.3 Setup

** NOTE **
You only need to run this script once after extracting the Zephyr SDK
distribution bundle archive.

Install host tools [Y,N]?Y
Register Zephyr SDK CMake package [Y,N]?Y

Installing host tools ...
SKIPPED: Windows host tools are not available yet.

Registering Zephyr SDK CMake package ...
Zephyr-sdk (C:/netfeasaproject/zephyr-sdk-0.16.3/cmake)
has been added to the user package registry in:
HKEY_CURRENT_USER\Software\Kitware\CMake\Packages\Zephyr-sdk


All done.

Press any key to exit ...
```

**Comment by skylinezone** [comment]
Did you soveld this problem bro? I meet this problem just now

**Reply by zafersn** [reply]
it is not actually an issue :(  you can ignore it if you are using windows

**Comment by skylinezone** [comment]
Oh I see thanks, but I found that when I run the examples. It says that no Zephyr SDK is installed  ---- Replied Message ***@***.***>Date11/16/2023 16:07 ***@***.***> CcZhang ***@***.***>***@***.***>SubjectRe: [zephyrproject-rtos/zephyr] SKIPPED: Windows host tools are not available yet. (Discussion #63914)
it is not actually an issue :(  you can ignore it if you are using windows

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/zephyrproject-rtos/zephyr/discussions/63914#discussioncomment-7585319",
"url": "https://github.com/zephyrproject-rtos/zephyr/discussions/63914#discussioncomment-7585319",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Reply by zafersn** [reply]
did you set up your workspace by following this instruction? https://docs.zephyrproject.org/latest/develop/getting_started/index.html
you can compare the steps you did to setup Zephyr toolchain etc..
This instruction worked for me  without problems
